index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title id="title">Microverse Trials</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id="title">Survey form</h1>
+    <p id="description">This is a form</p>
+  <div class="container">
+    <form id="survey-form">
+      <div class="group">
+        <label for="name" id="name-label"> Name <br>
+        <input type="text" id="name" placeholder="Enter your name here" class="size">
+        </label>
+      </div>
+      <div class="group">
+        <label for="email" id="email-label"> Email <br>
+        <input type="email" id="email" placeholder="Enter your email here" class="size">
+        </label>
+      </div>
+      <div class="group">
+        <label for="number" id="number-label"> Age <br>
+        <input type="number" id="number" min="18" max="3000" placeholder="Enter your age" class="size">
+        </label>
+      </div>
+      <div class="group">
```

Comment:
You could try to use fewer divs and leave the positioning for CSS, it's a good way to train your CSS skills.